source/howto/pipelines/pipeline-utils.rst

```diff
@@ -0,0 +1,96 @@
+.. meta::
+    :description: Creating pipelines with valohai-utils helper library.
+
+.. _pipeline-utils:
+
+Create pipelines by using valohai-utils
+#######################################
+
+In addition to creating pipelines in the web UI, it is also possible to use the `valohai-utils` helper library to do that. 
```

Comment:
```suggestion
It's possible to define pipelines using the ``valohai-utils`` helper library, in addition to manually defining them in the ``valohai.yaml`` configuration file.
```

Reply:
I'd focus on where you define the pipelines vs. where you launch them from. As you can't define a pipeline in the UI I wouldn't mention there here